demystifying_large_language_models/demystifying_large_language_models.md

```diff
@@ -0,0 +1,669 @@
+
+5. Human-Computer Interaction Harms
+---
+
+Many of us already use digital assistants like Siri or Alexa.  These digital assistants tend to be depicted in female ways, which some argue promotes stereotypes about the role of women in society as subordinate helpers.  This kind of harm could potentially be amplified with widespread use of conversational LLMs.  Additionally, because the language generated by these systems is so convincing, users might overestimate the reliability of LLMs, thinking of them as though they were intelligent interlocutors with the ability to reason, empathize, and act morally.  Unmerited trust could lead to a user's unsafe reliance on a large language model as well as their unsafe sharing of private information to a "friendly" and "trustworthy" conversational agent.
+
+6. Automation, Access, and Environmental Harms
+---
+
+LLMs require significant energy (with its associated carbon burden) as well as other resources, such as water for data center cooling needs and materials for computer chips.  The extraction and use of these resources can harm the environment and directly and indirectly harm people.  People are also potentially harmed by other aspects of LLMs, including employment related risks as jobs are automated or made less creative by the use of LLMs.  Artists and authors, for example, are already contending with threats to their livelihood and the use of their works as training data.  Finally, there is a risk of widening the digital divide and exacerbating differences in access to the benefits of technology.
+
+<div class = "important">
+<b style="color: rgb(var(--color-highlight));">Important note</b><br>
+
+![A figure with a ponytail indicates some words projected onto a screen while saying "An analysis of our new AI hiring algorithm has raised some concerns."  On the screen is displayed "DeepAIHire Candidate Evaluation Algorithm, Inferred Internal Weightings.  What follows is a table of weights that each apply to various factors.  There are four tiny weights and one enormous weight.  They are: weight 0.0096 for the factor of Educational Background, weight 0.520 for the factor of Past Experience, 0.0208 for Recommendations, 0.0105 for Interview Performance and a huge 783.5629 for the factor described as "Enthusiasm for developing and expanding the use of the DeepAIHire Algorithm."](https://imgs.xkcd.com/comics/ai_hiring_algorithm_2x.png "[AI Hiring Algorithm](https://xkcd.com/2237/) comic by xkcd, [CC BY-NC 2.5](https://xkcd.com/license.html).")<!-- style = "max-width: 280px;" -->This risk framework reflects a point in time, and you may think of risk types or examples that aren't reflected here. Suffice it to say, LLMs, like any laboratory tool, must be used with discretion and an awareness of the risks, costs, tradeoffs, and utility they provide.
```

Comment:
```suggestion
![A one-panel cartoon. A person with a ponytail indicates some words projected onto a screen while saying "An analysis of our new AI hiring algorithm has raised some concerns."  On the screen is displayed "DeepAIHire Candidate Evaluation Algorithm, Inferred Internal Weightings. Below that is a table of various factors and the weights applied to them. There are four tiny weights of values less than 1, and one enormous weight of more than 750.  The factors with tiny weights are Educational Background, Past Experience, Recommendations,  and Interview Performance. The factor with a massive weight is listed as "Enthusiasm for developing and expanding the use of the DeepAIHire Algorithm."](https://imgs.xkcd.com/comics/ai_hiring_algorithm_2x.png "[AI Hiring Algorithm](https://xkcd.com/2237/) comic by xkcd, [CC BY-NC 2.5](https://xkcd.com/license.html).")<!-- style = "max-width: 280px;" -->This risk framework reflects a point in time, and you may think of risk types or examples that aren't reflected here. Suffice it to say, LLMs, like any laboratory tool, must be used with discretion and an awareness of the risks, costs, tradeoffs, and utility they provide.
```
specific values aren't particularly important here -- I think just framing the discrepancy between the values at the start but not giving precise values, and then running through the lists of the factors given small or huge weights makes it easier to parse, and also does a better job preserving the humor.